### DISCUSSION TITLE
https://github.com/wimdeblauwe/error-handling-spring-boot-starter

### DISCUSSION THREAD

**Discussion by ianmcnicholas** [wimdeblauwe/wimdeblauwe.com#2] [General]
https://github.com/wimdeblauwe/error-handling-spring-boot-starter 
We are trying to use this library to implement error handling and logging in our API.  The issue we have is that we want to log out 4xx exceptions at info level, and 5xx exceptions at error level, however, the library only allows for error level logging for all exceptions.
We understand that you can turn off logging on 4xx exceptions, but we would like to keep them but just change them to info level.
Is there a work around to this?
Thanks.

**Comment by namhokim** [comment]
@ianmcnicholas I created issue like yours. I agree with you, too.

https://github.com/wimdeblauwe/error-handling-spring-boot-starter/issues/57

**Comment by namhokim** [comment]
@ianmcnicholas This issue was solved.

ref. https://github.com/wimdeblauwe/error-handling-spring-boot-starter/issues/57 https://github.com/wimdeblauwe/error-handling-spring-boot-starter/issues/61

Release notes:
- https://github.com/wimdeblauwe/error-handling-spring-boot-starter/releases/tag/4.1.0
- https://github.com/wimdeblauwe/error-handling-spring-boot-starter/releases/tag/3.4.0
